README.md

```diff
@@ -161,23 +161,37 @@ The following environment variables can be used to configure Celery, but, other
 * Default: none
 * Celery option: `-A`/`--app`
 
-> **NOTE**: The following 3 environment variables are deprecated. They will continue to work for now but it is recommended that you set these values in your Django settings file rather.
-
-#### `CELERY_BROKER`:
-* Required: no
-* Default: none
-* Celery option: `-b`/`--broker`
-
-#### `CELERY_LOGLEVEL`:
-* Required: no
-* Default: none
-* Celery option: `-l`/`--loglevel`
-
-#### `CELERY_CONCURRENCY`:
-Note that by default Celery runs as many worker processes as there are processors. **We instead default to 1 worker process** here to ensure containers use a consistent and small amount of resources. If you need to run many worker processes, they should be in separate containers.
-* Required: no
-* Default: **1**
-* Celery option: `-c`/`--concurrency`
+<details>
```

Comment:
`<details>` is not natively supported by Markdown so we've gotta use inline HTML.

Reply:
Interesting.. the preview of the file suggests it'll be displayed collapsed. Which is quite cool!

Reply:
Found it here: https://github.com/dear-github/dear-github/issues/166